misk/src/main/kotlin/misk/wire/AbstractClientCommand.kt

```diff
@@ -0,0 +1,24 @@
+package misk.wire
```

Comment:
I'm not sure this is sufficient value-add to add at the misk-level - the only thing it's really doing is defining a parameter for passing in the URL. I think that should just be left to the applications.